src/app/components/faq-assistant/faq-assistant.component.scss

```diff
@@ -0,0 +1,61 @@
+.container {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  padding-top: 50px;
+
+  .defaultClass {
```

Comment:
Always try to avoid the nesting, it is very difficult to override, and this also lead to slow style tree creating! Try to hold the css flat as long as you can.

Reply:
But the same approach uses in scss style

Reply:
was applied